tests/framework/ROM/TimeSeries/SyntheticHistory/autocorrelation.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/ROM/TimeSeries/SyntheticHistory.AutoCorrelation</name>
+    <author>rinelson456</author>
+    <created>2021-04-19</created>
+    <classesTested>SupervisedLearning.SyntheticHistory,TSA.AutoCorrelation</classesTested>
+    <description>
+      Tests the SyntheticHistory ROM using only the AutoCorrelation TimeSeriesAnalyzer algorithm.
```

Comment:
See, this bothers me a bit because a characterizer shouldn't work in a SyntheticHistory ROM. Maybe we need that postprocessor.

Reply:
Yes, this was a problem that @rinelson456 and I talked about. I think that a PP would work well in this situation.